requirements.txt

```diff
@@ -24,7 +24,7 @@ lnurl==0.3.6
 markupsafe==2.0.1
 marshmallow==3.13.0
 outcome==1.1.0
-psycopg2==2.9.3
```

Comment:
We [recently switched](https://github.com/lnbits/lnbits-legend/pull/658) to psychopg2 (instead of the binary) since it's recommended by the devs. Could you check whether the build works with the new requirement?

Reply:
It worked. Now I'm building from `python:3.10-buster`.

Reply:
reverting back to 3.7, as there were some incompatibilities